README.md

```diff
@@ -2,46 +2,73 @@
 
 This readme contains the steps to launch the **CWorld UML Editor**. This is a terminal based program that allows users to create a Class Diagram with Relations, Classes, and Attributes. Once the program is running, type 'help' for a list of commands.
 
-# Steps for Running the Program
+# Setup Your Environment
+<B>The minimum required version of Python to run this program is 3.8. If your version of python, found in the section below, is less than that, please follow [this](https://www.python.org/downloads/) link to install a newer version. </B>
 
+## Check that Python is Installed
 
+### MacOS
+<ol>
+<li> In the top right corner of your screen, there will be a search bar, a magnifying glass, or both (depending on your version of Mac). Click that.
```

Comment:
This can be done by hitting "cmd + space". It opens up the search bar. 
Can we mention this as well?